packages/fullstack/fullstack.txt

```diff
@@ -47,15 +61,20 @@ jspdf
 jstransform
 keyframes
 kjhtml
+ladda
 launchnavigator
 lcovonly
+lement
```

Comment:
isn't this just `element` with the `e` missing?

Reply:
Sometime, when coding we use the HTMLElement tag which isn't recognize by your plugin.
The "lement" was a quick fix, if a better fix exist I can change it :)

Reply:
Do you have an example file where `HTMLElement` doesn't correctly get checked? It should be working.

Reply:
In an angular project inside a .ts file

![image](https://user-images.githubusercontent.com/44493964/60982269-14902d80-a338-11e9-840e-ad8b226e00a2.png)

Reply:
There is something else going on in your case. Do you have any `ignoreRegExpList` defined in your settings? Something that ignores all caps words?

Reply:
Well, it was the problem. Sorry for the inconvenience caused